lib/time.js

```diff
@@ -152,17 +152,27 @@ function CronTime(luxon) {
 			}
 
 			if (typeof this.utcOffset !== 'undefined') {
-				let offset =
-					this.utcOffset >= 60 || this.utcOffset <= -60
-						? this.utcOffset / 60
-						: this.utcOffset;
+				if (typeof this.utcOffset === 'string') {
+					const parts = this.utcOffset.split(':');
+					const hours = parseInt(parts[0]);
+					const minutes = parseInt(parts[1]);
+					if (this.utcOffset[0] === '-') this.utcOffset = hours * 60 - minutes;
+					else this.utcOffset = hours * 60 + minutes;
+				}
+				let offset = this.utcOffset / 60;
+
 				offset = parseInt(offset);
 
+				const remainderMins = Math.abs(this.utcOffset - offset * 60);
+				const remainderMinsString =
+					remainderMins >= 10 ? remainderMins : '0' + remainderMins;
+
```

Comment:
to improve naming consistency, I suggest renaming a few variables: 
- `offset` => `offsetHours`  
- `remainderMins` => `offsetMins`  
- `remainderMinsString` => `offsetMinsStr`